### PR TITLE
adding /execution/* REST API

### PR DESCRIPTION
**Pull request by DenysGonchar**
see changes in [swagger ui](https://esl.github.io/amoc_rest/?v=amoc_dist).
only `openapi.yaml` file to review, the rest can be skipped.
FYI:

```
409 Conflict
    Indicates that the request could not be processed because of conflict in the current state of the resource, such as an edit conflict between multiple simultaneous updates.
```
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -143,6 +116,116 @@ paths:
...
+      requestBody:
+        description: request body (as json)
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ExecutionStart'
+        required: true
+      responses:
+        '200':
+          description: execution started succesfully
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: execution started successfully
```

File: openapi.yaml [hunk 2]
```diff
@@ -143,6 +116,116 @@ paths:
...
+              schema:
+                $ref: '#/components/schemas/Error'
+  /execution/stop:
+    patch:
+      tags:
+        - execution
+      description: Stop scenario
+      responses:
+        '200':
+          description: execution stopped succesfully
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: execution stopped successfully
```

File: openapi.yaml [hunk 3]
```diff
@@ -143,6 +116,116 @@ paths:
...
+        '409':
+          description: scenario is not running
+        '500':
+          description: internal error occured
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: internal error occurred
```

File: openapi.yaml [hunk 4]
```diff
@@ -143,6 +116,116 @@ paths:
...
+            schema:
+              $ref: '#/components/schemas/ExecutionChangeUsers'
+        required: true
+      responses:
+        '200':
+          description: users added successfully
+        '409':
+          description: scenario is not running
+        '500':
+          description: internal error occured
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: internal error occurred
```

File: openapi.yaml [hunk 5]
```diff
@@ -143,6 +116,116 @@ paths:
...
+            schema:
+              $ref: '#/components/schemas/ExecutionChangeUsers'
+        required: true
+      responses:
+        '200':
+          description: users removed successfully
+        '409':
+          description: scenario is not running
+        '500':
+          description: internal error occured
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: internal error occurred
```

File: openapi.yaml [hunk 6]
```diff
@@ -143,6 +116,116 @@ paths:
...
+            schema:
+              $ref: '#/components/schemas/ExecutionUpdateSettings'
+        required: true
+      responses:
+        '200':
+          description: Settings updated successfully
+        '409':
+          description: scenario is not running
+        '500':
+          description: internal error occured
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: internal error occurred
```

File: openapi.yaml [hunk 7]
```diff
@@ -143,6 +116,116 @@ paths:
...
+        required: true
+      responses:
+        '200':
+          description: execution started succesfully
+        '409':
+          description: >-
+            execution cannot be started (e.g. another scenario is already
+            running)
+        '500':
+          description: internal error occured
```
Comment (NelsonVides) [inline openapi.yaml]:
```suggestion
          description: internal error occurred
```

**Review by NelsonVides — COMMENTED** [review]
Just a few language check points 🙂

File: openapi.yaml (hunk 7 above)
Comment (DenysGonchar) [reply openapi.yaml]: > Just a few language check points 🙂

damn copy/past :)

**Review by NelsonVides — APPROVED** [review]
Cool 😄

**Pull request merged**